### ISSUE TITLE
Cocoapods unable to read CocoaPods-version.yml

### ISSUE THREAD

**Issue by sgtsquiggs** [CocoaPods/CocoaPods#4436]
After updating cocoapods from 0.37.x to 0.39:

```
~/P/P/nara ❯❯❯ rm -rf ~/.cocoapods/repos/master                                                                                                                                                                                                              [feature/type_audit] ✭ ✖ ✱
~/P/P/nara ❯❯❯ pod setup --verbose                                                                                                                                                                                                                           [feature/type_audit] ✭ ✖ ✱

Setting up CocoaPods master repo

Creating shallow clone of spec repo `master` from `https://github.com/CocoaPods/Specs.git` (branch `master`)
  $ /usr/local/bin/git clone https://github.com/CocoaPods/Specs.git master --depth=1
  Cloning into 'master'...
  Checking out files: 100% (55586/55586), done.
  $ /usr/local/bin/git checkout master
  Already on 'master'
  Your branch is up-to-date with 'origin/master'.
[!] There was an error reading '/Users/matthew/.cocoapods/repos/master/CocoaPods-version.yml'.
Please consult http://blog.cocoapods.org/Repairing-Our-Broken-Specs-Repository/ for more information.

/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/sources_manager.rb:339:in `rescue in version_information'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/sources_manager.rb:336:in `version_information'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/sources_manager.rb:253:in `check_version_information'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command/repo/add.rb:45:in `block in run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command/repo/add.rb:41:in `run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command/setup.rb:84:in `add_master_repo'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command/setup.rb:40:in `block in run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command/setup.rb:32:in `run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Users/matthew/.rvm/gems/ruby-2.1.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/Users/matthew/.rvm/gems/ruby-2.1.0/bin/pod:23:in `load'
/Users/matthew/.rvm/gems/ruby-2.1.0/bin/pod:23:in `<main>'
/Users/matthew/.rvm/gems/ruby-2.1.0/bin/ruby_executable_hooks:15:in `eval'
/Users/matthew/.rvm/gems/ruby-2.1.0/bin/ruby_executable_hooks:15:in `<main>'
~/P/P/nara ❯❯❯                                                                                                                                                                                                                                             ⏎ [feature/type_audit] ✭ ✖ ✱
```


```
~/.c/r/master ❯❯❯ pwd                                                                                                                                                                                                                                                          [master]
/Users/matthew/.cocoapods/repos/master
~/.c/r/master ❯❯❯ cat CocoaPods-version.yml                                                                                                                                                                                                                                    [master]
---
min: 0.32.1
last: 0.39.0
~/.c/r/master ❯❯❯
```

**Comment by segiddins** [comment]
What's the output of `gem list psych`?

**Comment by sgtsquiggs** [comment]
The fix mentioned in #3539 fixed. I had to `gem update --system`. This is not intuitive.
